ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -590,6 +712,25 @@ export class VaultMaxiProgram extends CommonProgram {
           '% of your reinvest. Feel free to donate more manually'
         : 'Thank you for donating ' + this.settings.autoDonationPercentOfReinvest + '% of your rewards'
 
+    const reinvestMessage =
+      this.reinvestTargets.length > 0
+        ? 'reinvest Targets:\n  ' +
+          this.reinvestTargets
+            .map(
+              (target) =>
+                target.percent!.toFixed(1) +
+                '% ' +
+                (target.targetAddress !== undefined
+                  ? 'sending to ' + simplifyAddress(target.targetAddress)
+                  : target.isCollateral
+                  ? 'reinvesting'
+                  : 'swapping to wallet') +
+                ' as ' +
+                target.tokenName,
+            )
+            .reduce((a, b) => a + '\n  ' + b)
+        : 'no reinvest targets -> no reinvest'
```

Comment:
to make this more readable, use a new enum for each target which is mentioned above. just a suggestion